ynr/settings/base.py

```diff
@@ -501,3 +502,9 @@ def _is_running_tests():
     from .testing import *  # noqa
 else:
     RUNNING_TESTS = False
+
+# Allowlist of URLs that should be ignored by dc_utils BasicAuthMiddleware
+BASIC_AUTH_ALLOWLIST = [
+    "/api",
+    "/api/*",
```

Comment:
Being an EC2-based app, YNR does have a healthcheck endpoint.

Annoyingly it is just `/` rather than a dedicated endpoint so we would need to allow that but if we deployed this as-is, we'd go into a loop of:

- bring up a new instance
- the load balancer decides it is unhealthy and shuts it down
- bring up a new instance

..and so on.

Given we are probably parking this until we actually have a staging site, maybe lets not not bogged down but I did just want to note that here.